db/seed_helper.rb

```diff
@@ -5,4 +5,15 @@ def create_roles_and_permissions
   Role.create!(:name => Role::DATA_OWNER_ROLE)
 end
 
-
+# Populates the languages table with the language names and codes from the languages CSV file
+def populate_languages
+  Language.delete_all
```

Comment:
The Language is essentially just being used as a storage/container for populating a dropdown and doesn't have any associations with other models. Languages would also only be created from the CSV instead of from user input/interaction, so there shouldn't be any data loss by deleting all of them (since it is re-imported from the CSV straight after deletion).